src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -796,7 +796,10 @@ unique_ptr<Scene> svgSceneBuild(SvgNode* node, float& vx, float& vy, float& vw,
 
     Box vBox = {vx, vy, vw, vh};
     auto docNode = _sceneBuildHelper(node, vBox, svgPath, false, 0);
+
     _applySvgViewFlag(docNode.get(), vx, vy, vw, vh, w, h, viewFlag);
+    w *= node->node.doc.sw;
```

Comment:
So in this case, you concered no given Viewbox, no given pt ViewSize but only Pencentage View SIze.
Does it the SVG standard or just you idea? Otherwise we could ignore the scaling here?

Reply:
I'm not 100% percent... we had some time ago discussion about this with no conclusions actually. but we need proper width and height for gradient in case userSpaceOnUse is set. and this way it works